tests/rules/test.rdf.js

```diff
@@ -14,7 +14,7 @@ describe('RDF: mustNotExist', () => {
 
     return rdfScanner.getContents()
       .then((xmlDoc) => {
-        return rules.mustNotExist(xmlDoc, rdfScanner.namespace);
+        return rules.mustNotExist(xmlDoc, null);
```

Comment:
Elsewhere we've used a dummy filename. Make it a bit easier to understand what this is.